### PR TITLE
Fix BR etcd backup and restore page

### PR DESCRIPTION
**Pull request by jiayiwang7**
*Issue #, if available:*

Fix comments that some steps need to be executed on etcd nodes instead of control plane nodes.
Also use `etcd` instead of `ETCD` for doc consistency.

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/etcd-backup-restore/bottlerocket-etcd-backup.md [hunk 1]
```diff
@@ -141,37 +141,37 @@ Make sure to setup the [admin environment variables]({{< relref "#admin-machine-
...
-1. Perform the ETCD restore
+1. Perform the etcd restore
     
-    For this step, you have to SSH into each ETCD node and run the restore command.
-    - Get ETCD nodes IPs for SSH'ing into the nodes
+    For this step, you have to SSH into each etcd node and run the restore command.
+    - Get etcd nodes IPs for SSH'ing into the nodes
     ```bash
-    # This should print out all the control plane IPs
+    # This should print out all the etcd IPs
```
Comment (jiayiwang7) [inline docs/content/en/docs/clustermgmt/etcd-backup-restore/bottlerocket-etcd-backup.md:154]: @abhinavmpandey08

**Review by abhinavmpandey08 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**